### DISCUSSION TITLE
Welcome to Samarium Discussions!

### DISCUSSION THREAD

**Discussion by shyamsitaula** [shyamsitaula/samarium#112] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mirondavide** [comment]
👋 Hi everyone!
My name is Davide Miron, I’m a 17-year-old student from Italy.
I’m passionate about front-end development and I’m currently learning Flutter.
I recently built [conferintaeuropeana.it](https://conferintaeuropeana.it/), a website for a European conference.
